### PR TITLE
Tag DataArrays.jl v0.5.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaStats/DataArrays.jl](https://github.com/JuliaStats/DataArrays.jl)
Release: [v0.5.0](https://github.com/JuliaStats/DataArrays.jl/releases/tag/v0.5.0)
Diff: [vs v0.4.0](https://github.com/JuliaStats/DataArrays.jl/compare/2d3d6e0606e62a7e267c7974592fcd5e7da05e4d...36be3152e1ff785ce4015c0218a3a2c29c3e3465)
`requires` vs v0.4.0: 
```diff
--- 0.4.0/requires
+++ 0.5.0/requires
@@ -1,5 +1,5 @@
 julia 0.6-
 Compat 0.8.6
-StatsBase 0.3
+StatsBase 0.14.1
 Reexport
 SpecialFunctions
```
cc: @ararslan

### CODE REVIEW

File: DataArrays/versions/0.5.0/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.6-
+Compat 0.8.6
+StatsBase 0.14.1
```
Comment (tkelman) [inline DataArrays/versions/0.5.0/requires]: https://github.com/JuliaStats/StatsBase.jl/commit/f6b3c0ef9cd0881cc4d2d809f306bc4ffa8a97f8 was 0.14.0 wasn't it?

Comment (ararslan) [inline DataArrays/versions/0.5.0/requires]: Yep. Not sure why that was changed to 0.14.1. I'll fix that and retag.

**Comment by tkelman** [comment]
Several failures caused by this:
- https://github.com/JuliaCI/pkg.julialang.org/blob/9ecb4c01a1d3d89f46205b4aa1cdc0f6a5f5c872/logs/Bootstrap_0.6.log
- https://github.com/JuliaCI/pkg.julialang.org/blob/9ecb4c01a1d3d89f46205b4aa1cdc0f6a5f5c872/logs/DataFrames_0.6.log
- https://github.com/JuliaCI/pkg.julialang.org/blob/9ecb4c01a1d3d89f46205b4aa1cdc0f6a5f5c872/logs/ExcelReaders_0.6.log
- https://github.com/JuliaCI/pkg.julialang.org/blob/9ecb4c01a1d3d89f46205b4aa1cdc0f6a5f5c872/logs/FixedEffectModels_0.6.log
- https://github.com/JuliaCI/pkg.julialang.org/blob/9ecb4c01a1d3d89f46205b4aa1cdc0f6a5f5c872/logs/GLM_0.6.log
- https://github.com/JuliaCI/pkg.julialang.org/blob/9ecb4c01a1d3d89f46205b4aa1cdc0f6a5f5c872/logs/MLDataUtils_0.6.log
- https://github.com/JuliaCI/pkg.julialang.org/blob/9ecb4c01a1d3d89f46205b4aa1cdc0f6a5f5c872/logs/ScikitLearn_0.6.log

**Comment by nalimilan** [comment]
Possible cause is https://github.com/JuliaStats/DataArrays.jl/pull/243.

**Comment by tkelman** [comment]
worth bisecting on DataArrays

**Comment by nalimilan** [comment]
I've got a fix at https://github.com/JuliaStats/DataArrays.jl/pull/251.

**Pull request merged**